vts/plugins/scheme-cca-ssd-platform/main.go

```diff
@@ -95,7 +96,7 @@ func (s Scheme) SynthKeysFromRefValue(
 	}
 
 	finalstr := ccaReferenceLookupKey(tenantID, implID)
-	log.Printf("CCA Plugin CCA Reference Value Look Up Key= %s\n", finalstr)
+	log.Debug("CCA Plugin CCA Reference Value Look Up Key= %s\n", finalstr)
 
 	return []string{ccaReferenceLookupKey(tenantID, implID)}, nil
```

Comment:
why not reusing `finalstr` instead of recomputing the value?

Besides, I suggest changing `finalstr` to something more meaningful, e.g.: `lookupKey`